packages/core/src/http-middlewares/after/prepare-response.js

```diff
@@ -29,20 +31,23 @@ const prepareContentResponse = (resp, request) => {
   // TODO: does it make sense to not trim the signature? more equivalence to raw...
   return resp.text().then((content) => {
     // trim down the response signature a ton for simplicity
-    let json;
-    try {
-      json = JSON.parse(content);
-    } catch (err) {
-      json = undefined;
-    }
     const preppedResp = {
       status: resp.status,
-      json: json,
+      json: undefined,
+      data: undefined,
       content: content,
       request: request,
       skipThrowForStatus: request.skipThrowForStatus,
     };
     const outResp = _.extend(preppedResp, replaceHeaders(resp));
+    try {
+      if (outResp.headers.get('content-type') === FORM_TYPE) {
+        outResp.data = querystring.parse(content);
+      } else {
+        outResp.data = JSON.parse(content);
+        outResp.json = outResp.data; // DEPRECATED
```

Comment:
Keeps `.json` around for now. As v10 is breaking we could also drop it. We also may want to not make it a reference to `.data` to prevent unexpected side-effects, but I don't think it's a big deal.

Reply:
Yeah. it's _probably_ fine to have them be a reference to each other, but we also don't lose much from making them a copy. I'd probably replace w/ another call to `JSON.parse`